lighthouse-cli/bin.ts

```diff
@@ -114,6 +116,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', 25)
```

Comment:
main benefit of having it here is the `--help` printing, but if we want a default it has to happen internally as well for anyone using lighthouse as a module.

What about exposing on Driver as a constant, so it would still be single source of truth `.default('timeout', Driver.MAX_WAIT_FOR_FULLY_LOADED)`?